js/ui/discoveryFeedButton.js

```diff
@@ -63,25 +56,36 @@ const DiscoveryFeedButton = new Lang.Class({
         this.parent({ vertical: true,
                       style_class: 'discovery-feed-button' });
 
-        this.bar = new St.Button({ name: 'discovery-feed-bar',
-                                   child: new St.Icon({ style_class: 'discovery-feed-bar-icon' }),
-                                   style_class: 'discovery-feed-bar',
-                                   visible: _primaryMonitorWidthPassesThreshold() });
-        this.add(this.bar);
+        this._bar = new St.Button({ name: 'discovery-feed-bar',
+                                    child: new St.Icon({ style_class: 'discovery-feed-bar-icon' }),
+                                    style_class: 'discovery-feed-bar',
+                                    visible: _primaryMonitorWidthPassesThreshold() });
```

Comment:
Some comment applies to this leading space added here, it could have been aded in the previous commit too